package.json

```diff
@@ -132,7 +132,7 @@
     "vite": "5.1.3",
     "vite-tsconfig-paths": "^4.2.2",
     "wait-on": "^7.0.1",
-    "wrangler": "^3.24.0"
+    "wrangler": "^3.72.3"
```

Comment:
Unrelated to this PR, our CI integration suite was failing due to outdated `wrangler` version, so batched that fix in here.